src/config.rs

```diff
@@ -186,12 +187,12 @@ impl Config {
             return 0;
         }
         if self.enable_log_recycle && self.purge_threshold.0 >= self.target_file_size.0 {
-            // (1) At most u32::MAX so that the file number can be capped into an u32
-            // without colliding. (2) Add some more file as an additional buffer to
-            // avoid jitters.
+            // (1) At most MAX_RECYCLE_CAPACITY so that the file number can be capped into
```

Comment:
Why have a limit?

Reply:
Concerns on exceeding the limitation of open files when testing. It has been revoked with refactoring the related test case.